_episodes/10-data-transformation.md

```diff
@@ -38,9 +38,67 @@ So far we've been looking only at 'String' type data. Much of the time it is pos
 >3. Note how the values are now displayed in green and follow a standard convention for their display format (ISO 8601) - this indicates they are now stored as date data types in OpenRefine. We can now carry out functions that are specific to Dates
 >4. On the Date column dropdown select ```Edit column->Add column based on this column```. Using this function you can create a new column, while preserving the old column
 >5. In the 'New column name' type "Formatted-Date"
->6. In the 'Expression' box type the GREL expression ```value.toString("dd MMMM yyyy")```
+>6. In the 'Expression' box type the GREL expression ```value.toString("dd MMM yyyy")
 {: .checklist}
 
+>## Understanding GREL Expressions
+>
+>General Refine Expression Language (GREL) allows to manipulate data in OpenRefine.
```

Comment:
GREL has already been explained in lessons 7 and 8, so I don't think we need to repeat here - this is specifically about specifying date formatting in GREL expressions